src/popup/router/components/InviteItem.vue

```diff
@@ -1,36 +1,58 @@
 <template>
   <div class="invite-row">
     <div class="invite-info">
-      <span class="primary">{{ balance }} {{ $t('pages.appVUE.aeid') }}</span>
+      <span class="primary">
+        <span class="balance">{{ balance }}</span>
```

Comment:
`.balance` is a global class, I think we should consider them as deprecated and avoid their usage.